docs/integrations/xgboost.rst

```diff
@@ -0,0 +1,189 @@
+Automatic logging of the XGBoost training
+=========================================
+.. image:: ../_static/images/xgboost/xgboost_0.png
+   :target: ../_static/images/xgboost/xgboost_0.png
+   :alt: XGBoost overview
+
+XGBoost is an optimized distributed gradient boosting library designed to be highly efficient, flexible and portable. Integration with |Neptune| lets you log multiple training artifacts out-of-the-box. It is implemented as XGBoost callback and includes:
+
+* |metrics| (train and eval) after each boosting iteration.
+* |model| (Booster) to Neptune after last boosting iteration.
+* |feature| to Neptune as image after last boosting iteration.
+* |tree| to Neptune as images after last boosting iteration.
+
+.. tip:: Try integration right away with this |google-colab|.
+
+Prerequisites
+-------------
+This integration makes use of the XGBoost library and is part of the |neptune-contrib|.
+
+Make sure you have all dependencies installed. You can use bash command below:
+
+.. code-block:: bash
+
+    pip install neptune-client pandas xgboost scikit-learn matplotlib graphviz neptune-contrib[monitoring]
```

Comment:
hmm, aren't those libs in `neptune-contrib['monitoring']`?
If not, I feel they probably should be, because otherwise, it may be error-prone for users that don't know it and everything goes smoothly up until the last tree logging with graphviz that is not present. 
I feel those libs should be inside of monitoring (xgboost doesn't have to).

Reply:
Hey,
Some of them are, some of them are not. I put everything here for clarity. User see it and know all dependencies.

BTW: xgboost does not have matplotlib, graphviz in their dependencies. If you install xgboost and try to log tree it end up with an error.

Anyways, I'll put them as monitoring dependency.